ext/openssl/tests/openssl_get_cert_locations_basic.phpt

```diff
@@ -0,0 +1,11 @@
+--TEST--
+openssl_get_cert_locations() tests
+--SKIPIF--
+<?php if (!extension_loaded("openssl")) print "skip"; ?>
+--FILE--
+<?php
+// openssl locations differ per distro.
+var_dump(count(openssl_get_cert_locations()));
+?>
+--EXPECTF--
+int(8)
```

Comment:
I'd suggest to test that the correct keys are returned with string values and that ini for cafile and capath can be set.